dev/provider_packages/prepare_provider_packages.py

```diff
@@ -2072,14 +2075,9 @@ def summarise_total_vs_bad_and_warnings(total: int, bad: int, warns: List[warnin
     ),
     (
         "Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since"
-        " Python 3.3, and in 3.10 it will stop working",
+        " Python 3.3,and in 3.9 it will stop working",
```

Comment:
```suggestion
        " Python 3.3, and in 3.9 it will stop working",
```

Reply:
Oh, I see, never mind.

Reply:
Yeah. The typo is in the warning. I was also surprised :)